extension.js

```diff
@@ -12,40 +12,66 @@ const activate = () => {
       return new Promise((resolve, reject) => {
         const luaFormatterScriptDir = __dirname + '/luacode';
         const luaFormatterScript = __dirname + '/luacode/formatter.lua';
-        const luaPath = vscode.workspace.getConfiguration('vscode-metalua-formatter').get('luaPath');
         const indentSize = vscode.workspace.getConfiguration('vscode-metalua-formatter').get('indentSize');
+        const platform = os.platform();
+        const arch = os.arch();
+        let builtInLuaPath;
 
-        if(!luaPath) {
-          vscode.window.showErrorMessage('Lua 5.1 path has not been specified in the configurations. Make sure to fill the \'luaPath\' field in the extensions settings.');
+        if(platform === "darwin") {
+          if(arch === "x64") {
+            builtInLuaPath = path.join(__dirname, 'lua', 'macos', 'lua51');
+          }
+          else if(arch === "arm64") {
+            builtInLuaPath = path.join(__dirname, 'lua', 'macos-arm64', 'lua51');
+          }
+        }
+        else if(platform === "linux") {
+          if(arch === "x64") {
+            builtInLuaPath = path.join(__dirname, 'lua', 'linux', 'lua51');
+          }
+          else if(arch === "arm64") {
+            builtInLuaPath = path.join(__dirname, 'lua', 'linux-arm64', 'lua51');
+          }
+        }
+        else if(platform === "win32") {
```

Comment:
Does it say this if its a 64 bit computer?

Reply:
Is there such thing as 32-bit Windows anymore?

Reply:
Apparently there is 32-bit Windows 10 but there's no 32-bit Windows 11. For this I'm okay with ignoring the existence of 32-bit Windows if this works on 64-bit Windows.

Reply:
According to the docs, the only Windows platform reported is `win32`